actions/dependabot-automerge/review_pr.py

```diff
@@ -24,63 +24,83 @@ def set_output(outputs):
             f.write(f"{name}={value}\n")
 
 
-def parse_semver_level(ctx, option, value: str):
-    return SemverLevel(value.removeprefix(UPDATE_TYPE_PREFIX))
+def parse_semver_update_type(ctx, option, value):
+    return SemverUpdateType(value)
 
 
-def parse_dependency_type(ctx, option, value: str):
-    return DependencyType(value.removeprefix(DEPENDENCY_TYPE_PREFIX))
+def parse_dependency_type(ctx, option, value):
+    return DependencyType(value)
 
 
 def parse_semver_autoapprovals(ctx, option, value):
     try:
-        return [SemverLevel(v.strip()) for v in value.split(",")]
+        return [
+            SemverUpdateType(SEMVER_UPDATE_TYPE_PREFIX + v.strip())
+            for v in value.split(",")
+        ]
     except ValueError:
-        raise click.BadParameter(
-            f"format must be a comma-separated list of semver values, e.g. {','.join([e.value for e in SemverLevel])}"
+        message = (
+            "format must be a comma-separated list of semver values, e.g. "
+            f"{','.join([e.name for e in SemverUpdateType])}"
         )
+        raise click.BadParameter(message)
 
 
-approval_message_template = "The {semver_level} update of this {dependency_type} dependency was automatically approved."
-denial_message_template = "The {semver_level} update of this {dependency_type} dependency was not automatically approved. For {dependency_type} dependencies, these semver updates can be automatically approved: {auto_approvals}"
+approval_message_template = (
+    "The `{semver_level}` update of this `{dependency_type}` "
+    "dependency was automatically approved."
+)
+denial_message_template = (
+    "The `{semver_level}` update of this `{dependency_type}` "
+    "dependency was not automatically approved. For `{dependency_type}` dependencies, "
+    "these semver updates can be automatically approved: {auto_approvals}"
+)
 
 
 def review_pr(
     *,
     dependency_type: DependencyType,
-    semver_level: SemverLevel,
-    prod_semver_autoapprovals: list[SemverLevel],
-    dev_semver_autoapprovals: list[SemverLevel],
+    semver_level: SemverUpdateType,
+    prod_semver_autoapprovals: list[SemverUpdateType],
+    dev_semver_autoapprovals: list[SemverUpdateType],
 ) -> tuple[bool, str]:
+    if dependency_type == DependencyType.indirect:
+        message = (
+            "This `indirect` dependency was not automatically approved. Only `direct` "
+            "dependencies may be automatically approved."
+        )
```

Comment:
Maybe add a code comment with this comment response?

https://github.com/mozilla/syseng-pod/commit/8723514473cfcb205b36b5befeb1aa4fbf9dda30#r122963348